tools/syz-testbed/testbed.go

```diff
@@ -82,150 +78,220 @@ func main() {
 		tool.Failf("failed to parse manager config: %s", err)
 	}
 	if managerCfg.HTTP == "" {
-		managerCfg.HTTP = ":50000"
+		managerCfg.HTTP = "0.0.0.0:0"
```

Comment:
You also removed increasePort, will it now work with anything other than 0 port? Or users should only override the IP address but keep 0 port?

Reply:
Manual port assignment turned out to be somewhat troublesome. We would also have to set RPC ports - otherwise randomly taken free ports sometimes overlap with the ports that were assigned to syz-managers. I've seen this happening a lot in the currently upstreamed version of the tool, and now that the tool periodically restarts the managers, this would happen more and more often.

Also, the experience has shown that there's actually no need to access individual dashboards - therefore we don't have to care about the ports at all.

Regarding this particular changed line - probably it'd be better just to always override HTTP with ":0".

Reply:
Some users may want to use "localhost:0", so I think it's fine as it is now.